README.md

```diff
@@ -4,38 +4,44 @@
 
 Azure Provider for the [Notation CLI](https://github.com/notaryproject/notation)
 
-The notation-azure-kv plugin provides the capability to sign the Notation generated payload by using Azure Key Vault (AKV). The user's certificate and private key should be stored in AKV and the plugin will request signing and getting the leaf certificate from AKV. 
+The `notation-azure-kv` plugin provides the capability to sign the Notation generated payload by using Azure Key Vault (AKV). The user's certificate and private key should be stored in AKV and the plugin will request signing and getting the leaf certificate from AKV. 
 
 The plugin supports Azure CLI identity and Managed Identity for accessing AKV.
 
 ## Installation
 Before installing notation azure key vault plugin, please make sure Notation CLI has been installed. If you didn't install it, please follow the [Notation installation guide](https://notaryproject.dev/docs/installation/cli/).
 
-1. Navigate to the [Releases](https://github.com/Azure/notation-azure-kv/releases) page and select the latest release of notation-azure-kv. The notation-azure-kv binaries for each platform are available under the Assets section. Please download it based on your own platform. 
+1. Navigate to the [Releases](https://github.com/Azure/notation-azure-kv/releases) page and select the latest release of `notation-azure-kv`. The `notation-azure-kv` binaries for each platform are available under the Assets section. Please download it based on your own platform. 
 2. Validate the checksum which should be the same as the one in `checksums.txt` and then install the plugin.
    
    For Linux Bash:
    ```sh
+   version=0.6.0
+
    # validate checksum
-   sha256sum notation-azure-kv_<version>_linux_amd64.tar.gz
+   sha256sum notation-azure-kv_${version}_linux_amd64.tar.gz
```

Comment:
Comparing the hash value with `checksum.txt` should not be done by human eyes.

Think about
```sh
cat checksums.txt | grep notation-azure-kv_${version}_linux_amd64.tar.gz | sha256sum -c
```

Reply:
Updated.